src/common/config/process-killer-options.ts

```diff
@@ -0,0 +1,21 @@
+import { isWindows } from "../helpers/operating-system-helpers";
+import { platform } from "os";
+
+export interface ProcessKillerOptions {
+    isEnabled: boolean;
+    prefix: string;
+}
+
+const defaultMacOsProcessKillerOptions: ProcessKillerOptions = {
+    isEnabled: true,
+    prefix: "kill?",
+};
+
+const defaultWindowsProcessKillerOptions: ProcessKillerOptions = {
+    isEnabled: true,
+    prefix: "kill?",
+};
```

Comment:
These options are identical, so why separating them?

Reply:
Idk, I just copied them from another plugin plus I wanted to support both os but I can't do that not now at least,  I will change them.